olumidae/QuickCredit
# #166204302 Ensures user record is updated during signin

**What does this PR do?**
Ensures user signin records go to the database

**Description of the task to be completed?**
Setting up user sign in records into Postgres is functional
Updating isLogged column in the database to true on signin 

**How should this be manually tested?**
Clone the repo cd into it and run,
Using postman, test the POST /api/v1/auth/signin endpoint
Insert the required records and check the database in Postgres and check the islogged in column

**Any background context you want to provide?**
Not applicable

**What are the relevant pivotal tracker stories?**
#166204302
